GOVERNANCE.md

```diff
@@ -1,18 +1,97 @@
+* Truly open - as little restrictions as possible for anyone to adapt the Standard for Public Code to their context
+* Welcoming and respectful - as a community we want to make it easier for new users to become contributors.
+* Transparent and accessible, changes to the governance, Standard for Public Code code repository, and Standard for Public Code related activities (e.g. level, involvement, etc) are done in public
+* Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles
+
+## Steering team
+
+The community of Standard for Public Code has one steering team.
+
+### Composition
+
+Any active contributor in the community can request to become a steering team member by asking the steering team. The steering team will vote on it (see voting below).
+
+The current team members are:
+
+* Claus Mullie
+* Johan Groenen (Tiltshift, Code for NL)
+
+Ideally, no one organization will employ a majority of the steering team.
+
+### Responsibilities
+
+The steering team members are active contributors who are on a day-to-day basis responsible for:
+
+* Merging pull requests
+* Handling code of conduct violations
+
+Besides the day-to-day activities, the steering team has the joint responsibility to:
+
+* Provide technical direction for the codebase
+* Maintain a roadmap, and contributing principles
+* Resolve issues in development or conflicts between contributors
+* Managing and planning releases
+* Controlling access rights to Standard for Public Code assets such as source repositories, hosting and project calendars
+* Maintaining the mission, vision, values, and scope of the project
+* Refining the governance as needed
+* Making codebase level decisions
+* Managing the Standard for Public Code brand
+* Licensing and intellectual property changes
+
+### Meetings
+
+The steering team meets regularly.
+Their agenda includes review of the roadmap and issues that are at an impasse.
+The intention of the agenda is not to review or approve all patches.
+(That is mainly being done through the process described in [CONTRIBUTING.md](CONTRIBUTING.md).)
+
+## Decision making process
+
+The decision making process is lazy consensus as a default and voting for certain matters.
+
+### Lazy consensus
+
+For this community, lazy consensus means that if you think that a decision is uncontroversial you can just go ahead and make that decision.
+Any decision made this way is considered supported as long as no one objects.
+Of course, you have to be prepared to roll back your work if someone does object.
+(While the Apache community refers to this as [lazy consensus](https://community.apache.org/committers/decisionMaking.html), some communities refers to this as a consent model.)
+
+If there is uncertainty about a decision, a steering team member can inform the rest of the team that they are about to take a certain decision under lazy consensus.
```

Comment:
For me, this process is [precisely defined](https://circleforward.us/consent-vs-consensus-whats-the-difference/) as “consent”. I believe “consent” is a better wording since:

1. I saw significantly more literature calling this “consent” than “lazy consensus”.
2. I find the adjective “lazy” derogatory.
3. This is actually very different from “consensus”, so I believe it is better to reduce chances for confusion.

```suggestion
The decision making process is consent as a default and voting for certain matters.

### Consent

For this community, “consent” means that if you think that a decision is uncontroversial you can just go ahead and make that decision.
Any decision made this way is considered supported as long as no one objects.
Of course, you have to be prepared to roll back your work if someone does object.

If there is uncertainty about a decision, a steering team member can inform the rest of the team that they are about to take a certain decision.
```

Reply:
Agreed.

Curious whether anyone else feels strongly about this.

Reply:
"Lazy consensus is a **decision-making policy** which **assumes general consent** if no responses are posted within a defined period." (https://www.apache.org/foundation/glossary.html#LazyConsensus)

So I'd use "Lazy consensus", since it is the name of this policy, not just the words.

Reply:
I am not questioning it is defined, I just find this name very unusual (before this file, I had never heard of it), whereas “[consent vs consensus](https://duckduckgo.com/?q=consensus+vs+consent)” is a fairly well defined discussion. It sounds to me like this is Apache jargon. I am thus in favor of using a more common name to make it more likely that readers are familiar with it.
However, since this is just naming, this is not blocking for me.

Reply:
I propose going with consent then, as the apache definition is `assumes general consent`. I'll make these changes to unblock this, but I'm prepared to roll back if anyone feels strongly ;)